### ISSUE TITLE
Очень большая вложенность if

### ISSUE THREAD

**Issue by i-redbyte** [Vponed/Balancer#1]
```python
                            if (cpu_free < float(max_cpu) and max_cpu != '0'):
				logging('Загрузка cpu=' + str(cpu_free) + '<' + max_cpu)
				if (memory_free > max_ram):
					logging('Свободная память:mem=' + str(memory_free) + '>' + str(max_ram))
					if (upload < float(max_upload)) or max_upload == '0':
						logging('Upload (Mbps): ' + str(upload) + '<' + max_upload)
						if (download < float(max_download)) or max_download == '0':
							logging('Download (Mbps): ' + str(download) + '<' + max_download)
							if (flag_except_prog == False):
								logging("Запущена ли программа-исключение:" + str(flag_except_prog))
								logging('Все параметры в норме. Запускаем:' + exec_file)
								# , creationflags=subprocess.CREATE_BREAKAWAY_FROM_JOB
								success_var = subprocess.Popen(path_to_file + exec_file)
								logging("Успешно")
								pass
							pass
						pass
					pass
				pass
			pass
		
		pass
```

Когда большая вложенность if, то что-то пошло не так, предлагаю пересмотреть логику или же попробовать избавится  от избыточной вложенности путем пересмотра условий, например вместо
```python
if (cpu_free < float(max_cpu) and max_cpu != '0'):
```
написать, что-то типо

```python
if cpu_free >= float(max_cpu) or max_cpu == '0':
    continue
```

ну и тд

Но лучше подумать над общей структурой программы, разбить на ф-ции и тд

**Comment by Vponed** [comment]
Так-то все правильно, но смотри. идея проги отслеживать загрузку оборудования, и относительно нее вырубать ненужное. А при простое наоборот врубать.
Когда мы ВЫрубаем, никакой вложенности нет, достаточно ОДНОГО критического показателя. Выше по коду можешь глянуть, там ровно так, как ты написал. А вот когда мы ВРУБАЕМ, вот тогда ВСЕ указанные показатели должны быть в норме. Цель то сберечь ресурсы канпа. Например, указано вырубать при перегрузе процессора. Опция может быть отключена, проц может быть в норме, проц может быть не в норме - три варианта. И если проц в норме, это не значит, что не надо проверять память и проч.
Отсюда и вложенность. Надо все по очереди проверить. Но код грубый, согласен. Я немного поломал голову, но лучше не придумал. Класс с обработчиками написать - еще длиннее получится. Циклом перебрать параметры, тоже, больно сложный цикл. Читаемость и логика нарушится. Скорее, надо лучше прокомментировать)
Ваще так работает как часы). Ну и гитхаб больше как хранилище юзаю)
